pkg/kubelet/dockershim/network/cni/cni_windows.go

```diff
@@ -67,7 +67,16 @@ func (plugin *cniNetworkPlugin) GetPodNetworkStatus(namespace string, name strin
 		klog.Errorf("error while cni parsing result: %s", err)
 		return nil, err
 	}
-	return &network.PodNetworkStatus{IP: result020.IP4.IP.IP}, nil
+	 
+	var  list []net.IP;
+
+	list = append(list, result020.IP4.IP.IP)
```

Comment:
could be condensed to something like `var list = []net.IP{result020.IP4.IP.IP}`

Reply:
+1, the most frequent form is actually:

list := []net.IP{result020.IP4.IP.IP}

Reply:
Did you build this? You need to instantiate list. This should look something like 

`list := []net.IP{results020.IP4.IP.IP}` as Jocelyn said.